diesel/src/sqlite/expression/expression_methods.rs

```diff
@@ -0,0 +1,21 @@
+use expression::Expression;
+use super::operators::*;
+
+pub trait SqliteExpressionMethods: Sized {
+    /// Creates SQLite `COLLATE BINARY` expression
```

Comment:
Can you link to the sqlite docs in each of these comments so people who have no idea what that is have a link to click?